docs/guides/architecture/webviews.md

```diff
@@ -0,0 +1,159 @@
+[611.3.10.1.6]: https://github.com/WebKit/WebKit/blob/54099b931b220cf75dea154bb2e84a6a0582e87c/Source/WebKit/Configurations/Version.xcconfig
+[611.2.7.1.4]: https://github.com/WebKit/WebKit/blob/200180885a516f378d0253ffc7b950f98b3f9810/Source/WebKit/Configurations/Version.xcconfig
+[611.1.21.161.3]: https://github.com/WebKit/WebKit/blob/7aaa117b91a6822c40761d6f4da2e3d27627602f/Source/WebKit/Configurations/Version.xcconfig
+[610.2.11.1.3]: https://github.com/WebKit/WebKit/blob/f11e10bcbb474d8c65a870cc680b0964d6529748/Source/WebKit/Configurations/Version.xcconfig
+[610.2.11.51.8]: https://github.com/WebKit/WebKit/blob/388eae2d649eaecadaa11e1edc4248e54db583f7/Source/WebKit/Configurations/Version.xcconfig
+[611.1.21.1.12]: https://github.com/WebKit/WebKit/blob/5aebddad42f6572ffb20d1cd1be8d22be9cf0101/Source/WebKit/Configurations/Version.xcconfig
+[610.3.7.1.9]: https://github.com/WebKit/WebKit/blob/62e4387a5eab36ed075961d9ee9971f8c01a55bd/Source/WebKit/Configurations/Version.xcconfig
+[610.4.3.1.4]: https://github.com/WebKit/WebKit/blob/b152d7889c786689406f203cc4eefea509a90302/Source/WebKit/Configurations/Version.xcconfig
+[610.4.3.1.7]: https://github.com/WebKit/WebKit/blob/248c3283ebdec8bd8ae05d4d1d56390b0da28f27/Sour.3ce/WebKit/Configurations/Version.xcconfig
+[609.3.5.1.3]: https://github.com/WebKit/WebKit/blob/30fc8a44f087596c60e98adb434c0b98eccb61bb/Source/WebKit/Configurations/Version.xcconfig
+[609.4.1.1.1]: https://github.com/WebKit/WebKit/blob/8df64286794c38efa4697b7c24658cb85204a070/Source/WebKit/Configurations/Version.xcconfig
+[609.4.1]: https://github.com/WebKit/WebKit/blob/cb927e6151b5ef49c9ccfb13018f51471f8f1035/Source/WebKit/Configurations/Version.xcconfig
+[609.2.9.1.2]: https://github.com/WebKit/WebKit/blob/ca54d252f3416c3ec64f80a084cb5c4ff7ba24f1/Source/WebKit/Configurations/Version.xcconfig
+[609.1.20.111.8]: https://github.com/WebKit/WebKit/blob/5c90480a38a86464b6b421c2fd28c744b43a4faa/Source/WebKit/Configurations/Version.xcconfig
+
+## Linux (WebKitGTK)
+
+| Distro                                                         | webkitgtk | WebKit    | Safari Equivalent |
+| -------------------------------------------------------------- | --------- | --------- | ----------------- |
+| debian 9                                                       | 2.18      | [605.1.3] | 11.1              |
+| Ubuntu 18.04 (arm64, armhf, ppc64el, s390x)                    | 2.20      | [606.1.4] | 12.0              |
+| Ubuntu 20.04 (arm64, armhf, ppc64el, s390x)                    | 2.28      | [610.1.1] | 14.0              |
+| Ubuntu 18.04 (amd64, i386), centos 8 (non-stream)              | 2.32      | [612.1.6] | 15.0              |
+| debian 10                                                      | 2.34      | [613.1.1] | 15.4              |
+| Ubuntu 20.04 (amd64, i386), Ubuntu 22.04, debian 11, debian 12 | 2.36      | [614.1.6] | TP 140 (16.0)     |
```

Comment:
Okay now that i see it, i think something like having `debian 9` and `debian 9*` with * being "With all updates applied" would be kinda cool i guess?

Also funny story, what i mentioned packages.ubuntu.com would show (but didn't) in _distro_ vs _distro_-updates (the first one being the packages when the distro version was released and the second one is the updates to it), is indeed what's happening for non x86 archs.
In other words: `(amd64, i386)` == all updates applied - `(arm64, armhf, etc)` == original version.

Reply:
Is this comment still relevant? I added a note about the situation on Ubuntu 20.04 and older and the arches, but I have no idea how to get more info about Debian 9 🤔 feel free to add that info if you have it though 😄

Reply:
yep still relevant, gonna propose changes for a thing or 2 in a few.